db/migrate/20200322182816_create_jams.rb

```diff
@@ -0,0 +1,11 @@
+class CreateJams < ActiveRecord::Migration[6.0]
```

Comment:
depending on what we do with jams in the future, it might benefit from having a public id, like rooms, too but let's wait and see.